compute-budget/src/compute_budget_limits.rs

```diff
@@ -21,15 +34,26 @@ const MICRO_LAMPORTS_PER_LAMPORT: u64 = 1_000_000;
 
 /// The total accounts data a transaction can load is limited to 64MiB to not break
 /// anyone in Mainnet-beta today. It can be set by set_loaded_accounts_data_size_limit instruction
-pub const MAX_LOADED_ACCOUNTS_DATA_SIZE_BYTES: NonZeroU32 =
+#[cfg_attr(feature = "svm-internal", qualifiers(pub))]
+const MAX_LOADED_ACCOUNTS_DATA_SIZE_BYTES: NonZeroU32 =
     unsafe { NonZeroU32::new_unchecked(64 * 1024 * 1024) };
 
 #[derive(Clone, Copy, Debug, PartialEq, Eq)]
-pub struct ComputeBudgetLimits {
-    pub updated_heap_bytes: u32,
-    pub compute_unit_limit: u32,
-    pub compute_unit_price: u64,
-    pub loaded_accounts_bytes: NonZeroU32,
+#[cfg_attr(
+    any(feature = "dev-context-only-utils", feature = "svm-internal"),
+    qualifiers(pub),
+    field_qualifiers(
+        updated_heap_bytes(pub),
+        compute_unit_limit(pub),
+        compute_unit_price(pub),
+        loaded_accounts_bytes(pub)
+    )
+)]
+pub(crate) struct ComputeBudgetLimits {
```

Comment:
litesvm needs to read the fields in ComputeBudgetLimits, which it builds by calling `solana_compute_budget_instruction::process_compute_budget_instructions`